aineko/extras/connectors/http_poller.py

```diff
@@ -0,0 +1,264 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Extra module for connecting to an HTTP endpoint."""
+
+import json
+import time
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import requests
+from pydantic import BaseModel, field_validator
+
+from aineko import AbstractNode
+
+
+class ParamsHTTPPoller(BaseModel):
+    """Parameters for the HTTPPoller node.
+
+    Attributes:
+        timeout: The number of seconds to wait for the endpoint to
+            respond. Defaults to 10.
+        url: The URL to connect to.
+        headers: A dictionary of headers to send to the endpoint. Defaults
+            to None.
+        data: A dictionary of data to send to the endpoint. Defaults to None.
+        params: A dictionary, list of tuples, bytes, or file-like object to
+            send in the body of the request. Defaults to None.
+        json_: A JSON serializable Python object to send in the body of the
```

Comment:
is this `json_` because json is a keyword?

Reply:
yes